TABTestKit/Classes/Protocols/Element.swift

```diff
@@ -49,6 +49,8 @@ public extension Element {
 	
 	var label: String { return underlyingXCUIElement.label }
 	
+	var value: String? {return underlyingXCUIElement.value as? String}
```

Comment:
Some missing spacing around the bracket.
```suggestion
	var value: String? { return underlyingXCUIElement.value as? String }
```